posts/2022-12-14-android-ndk-update.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "Updating the Android NDK in Rust <version>"
+author: Android Platform Team
+description: "Modernizing Android support in Rust"
+---
+
+We are pleased to announce that Android platform support in Rust will be
+modernized in Rust <version> as we update the target NDK from r17 to r25.  As a
+consequence the minimum supported API level will increase from 15 (Jelly Bean)
```

Comment:
I believe 15 is Ice Cream Sandwich.

Reply:
It it ICS. https://en.wikipedia.org/wiki/Android_version_history is actually the easiest resource for this.

Reply:
Done